package.json

```diff
@@ -118,14 +122,12 @@
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged",
-      "pre-push": "make checkBindings",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
   "prettier": "@stencila/dev-config/prettier-config.json",
   "release": {
-    "extends": "@stencila/semantic-release-config",
-    "message": "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
```

Comment:
This custom message was required to remove `[skip ci]` so that the Python package gets released on a tagged commit. See the message for d9b93933bf76b789f2fd649b4448826db7cf2bfd. I think that is still the case, so probably best not to remove this?

Reply:
ah, darn it. Removed the changes once already, but snuck back in during rebase as the version of `dev-config` is outdated here.

Reply:
@nokome should be good for another round of 👀